project/plugins.sbt

```diff
@@ -6,7 +6,7 @@ resolvers += "Typesafe repository" at "https://repo.typesafe.com/typesafe/releas
 
 addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "1.1.18")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.2")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.8")
```

Comment:
It seems like this is the first version that's compatible with sbt `1.5.x` - see https://github.com/guardian/ophan/pull/4085 for more details.